apps/system-e2e/src/tests/islandis/application-system/acceptance/parental-leave.spec.ts

```diff
@@ -16,13 +20,51 @@ import {
   EmailAccount,
   makeEmailAccount,
 } from '../../../../support/email-account'
-import { helpers, locatorByRole } from '../../../../support/locator-helpers'
+import { helpers } from '../../../../support/locator-helpers'
 import { session } from '../../../../support/session'
-import { mockApi } from '../../../../support/api-tools'
 import { setupXroadMocks } from './setup-xroad.mocks'
 
 test.use({ baseURL: urls.islandisBaseUrl })
 
+async function getEmployerEmailAndApprove(employer: EmailAccount, page: Page) {
+  const { proceed } = helpers(page)
+
+  const email = await employer.getLastEmail(6)
+
+  if (email && typeof email.html === 'string') {
+    const employerUrlMatch = email.html.match(/>(http?:.*)<\/p>/)
+    if (employerUrlMatch?.length != 2)
+      throw new Error(
+        'Email does not contain the url to approve the parental leave application',
+      )
+    const employerUrl = employerUrlMatch[1]
+    if (!employerUrl)
+      throw new Error(`Could not find url for employer in email: ${email.html}`)
+    await page.goto(employerUrl)
```

Comment:
If you're expecting this to be an application, please verify it with `expect (page).toBeApplication()`